IronSoftware.Drawing/IronSoftware.Drawing.Common/Color.cs

```diff
@@ -4,33 +4,42 @@
 
 namespace IronSoftware.Drawing
 {
+    // <summary>
+    /// <para>A universally compatible Color for .NET 7 and .NET 6, .NET 5, .NET Core. Windows, NanoServer, IIS,  macOS, Mobile, Xamarin, iOS, Android, Google Compute, Azure, AWS and Linux compatibility.</para>
+    /// <para>Works nicely with popular Image Color such as <see cref="System.Drawing.Color"/>, <see cref="SkiaSharp.SKColor"/>, <see cref="SixLabors.ImageSharp.Color"/>, <see cref="Microsoft.Maui.Graphics.Color"/>.</para>
+    /// <para>Implicit casting means that using this class to input and output Color from public API's gives full compatibility to all Color type fully supported by Microsoft.</para>
```

Comment:
Change:
`    /// <para>Implicit casting means that using this class to input and output Color from public API's gives full compatibility to all Color type fully supported by Microsoft.</para>
`
to:
`    /// <para>Implicit casting means that using this class to input and output Color from public APIs gives full compatibility to all Color-types fully supported by Microsoft.</para>
`